assignment/scripts/loops-practice.js

```diff
@@ -14,17 +14,35 @@ for (let i=0; i<4; i++) {
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
 
+for (let i=0; i<5; i++) {  
```

Comment:
Whoops! 5 is not less than 5, so as written this will only count from 0 to 4.

```suggestion
for (let i = 0; i <= 5; i++) {  
```